dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/token/TokenValidator.java

```diff
@@ -195,66 +194,17 @@ protected void validateSubject() throws TokenValidationException {
      *
      */
     protected void validateIssuer() throws TokenValidationException {
-        String iss = getIssuer(oidcConfig);//oidcConfig.getProviderMetadata().getIssuer();
-        if (!iss.equals(this.issuer)) {
+        if (this.issuer != null && !issuerconfigured.equals(this.issuer)) {
```

Comment:
We need to check issuer configured against the issuer all the time, regardless that issuer received is null.

Reply:
done